packages/datadog-instrumentations/src/helpers/register.js

```diff
@@ -10,6 +10,14 @@ const log = require('../../../dd-trace/src/log')
 const { DD_TRACE_DISABLED_INSTRUMENTATIONS = '' } = process.env
 
 const hooks = require('./hooks')
+// for any node:* prefix's also add a hook for the package without the prefix
```

Comment:
Why not just add it explicitly to the list in `hooks.js`? It would also be easier to understand to the reader.

Reply:
I think @bengl concern was that it results in duplication. Maybe @bengl chime in again to let us know if you still feel strongly about it.

Reply:
It's not the same module name, so there is no duplication. Also, being explicit would allow handling the same thing for modules that are not built-in.

Reply:
There _is_ duplication as Node.js itself will create on instance shared with both names while userland could require or import it with either name and we will patch them separately for each name.

Reply:
There won't be double patching assuming that some logic is present to avoid it. This was discussed when we chatted offline but I'm not sure it was ever actually implemented.

Reply:
Also I'm not sure how auto-aliasing would fix the issue as we would still be patching both modules.

Reply:
It was implemented it was apart of the first commit which made use of a weakmap to disable double patching, but @bengl preferred the current approach since it doesn't require a weakMap and because it is a little cleaner since we don't have to explicitly add node:* specifiers to our hooks.js and plugins.js lists